centos-7-from-iso.json

```
@@ -12,7 +12,7 @@
       "http_directory": "http",
       "iso_checksum": "38d5d51d9d100fd73df031ffd6bd8b1297ce24660dc8c13a3b8b4534a4bd291c",
       "iso_checksum_type": "sha256",
-      "iso_url": "http://mirrors.mit.edu/centos/7/isos/x86_64/CentOS-7-x86_64-Minimal-1810.iso",
+      "iso_url": "http://mirror1.es.uci.edu/centos/7.6.1810/isos/x86_64/CentOS-7-x86_64-Minimal-1810.iso",
```

```suggestion
      "iso_url": "http://mirrors.mit.edu/centos/7.8.2003/isos/x86_64/CentOS-7-x86_64-Minimal-2003.iso",
```

Let's stick with the same mirror but pin it as you did, note that I bumped up to the latest version.

---

In order for that to work, I think we'll need to change the iso_checksum number as well. Do you know where I can find that?

---

Think I found it in sha256sum.txt file.